packages/dialtone-vue2/recipes/leftbar/contact_row/contact_row.vue

```diff
@@ -19,12 +19,19 @@
         :image-src="avatarSrc"
         :color="avatarColor"
         image-alt=""
-        :icon-name="iconName"
-        icon-size="200"
         size="sm"
         :seed="avatarSeed"
         :presence="avatarPresence"
-      />
+      >
+        <template
+          v-if="noInitials"
+          #icon
+        >
+          <dt-icon-user
+            :size="200"
```

Comment:
icon size prop expects a String, this might raise a warning IIRC.

```suggestion
            :size="'200'"
```